theme/bootstrap-override/bootstrap-italia/_megamenu.scss

```diff
@@ -5,6 +5,8 @@
       &.active {
         .dropdown {
           a.nav-link {
+            padding-right: 18px;
```

Comment:
no px

Reply:
Secondo me in questo specifico caso è giusto usare px perché tutte le spaziature laterali di bootstrap (padding di container-row-col) sono in px, quindi così si allinea sempre preciso